libs/async_cuda/src/cuda_target.cpp

```diff
@@ -11,9 +11,10 @@
 #include <hpx/async_cuda/target.hpp>
 #include <hpx/futures/traits/future_access.hpp>
 #include <hpx/modules/errors.hpp>
-#include <hpx/modules/memory.hpp>
+#if defined(HPX_HAVE_DISTRIBUTED_RUNTIME)
 #include <hpx/runtime/find_here.hpp>
 #include <hpx/runtime/naming/id_type_impl.hpp>
+#endif
 #include <hpx/runtime_fwd.hpp>
 #include <hpx/threading_base/thread_helpers.hpp>
 
```

Comment:
Should the `#if`/`#endif` section below be adapted to `HPX_HAVE_DISTRIBUTED_RUNTIME` as well?

Reply:
I'm not sure which 'section below' you are referring to. Could you please elaborate. thanks

Reply:
I'm referring to the `#if`/`#endif` starting at line 21. Github didn't allow me to add comments to unchanged code blocks.

Reply:
sorry - I was looking at the wrong file. I see it now. I think probably yes, but I didn't get any compilation problems, so I didn't look further at it. I will experiment tomorrow with it.

Reply:
I don't think it needs to be adapted. Serialization is available even with the distributed runtime off.